### PR TITLE
Part 3: Updating dev with designer's map branch

### PR DESCRIPTION
**Pull request by KoolieAid**
# Description:
Updates the dev branch with commits from the designers
Part 3

### How it works:
TBA

## Guidelines:
- [x] I have consulted my lead when making this request
- [X] I have followed merging practices
- [X] I have assigned the proper target branch this will merge to

### Features added:
- TBA
- None

### Features removed:
- None

### Potential Affected branches:
- dev
- None

### Potential Affected features:
- None

## Leads Only
- [ ] This merge request has been approved by all the leads in my department

### CODE REVIEW

**Review by LauSnchzz — CHANGES_REQUESTED** [review]
TO DELETE DUE TO ERRORS IN BUILDING THE GAME:
- Content\PauseMenu\BP_PlayerPauseDuplicate.uasset
- Content\InspectObject\BP_PlayerDuplicate.uasset
- Content\InspectObject\BP_PlayerControllerDuplicate.uasset

EXTRA NOTE:
Content/TheFall/Levels/UMaps/Boss_Maze.umap has malformed tag. Make sure this issue is fixed before merging back to dev.

File: Content/TheFall/Levels/UMaps/Boss_Maze.umap
Comment (KoolieAid) [inline Content/TheFall/Levels/UMaps/Boss_Maze.umap:1]: Awaiting on @chartimbreza to provide a reference for the blob.  
There was a merge conflict was not resolved but commited

**Comment by KoolieAid** [comment]
Also fixes #60, github wont permit more than 10 linked issues

**Review by LauSnchzz — APPROVED** [review]
ok na tapos na

**Pull request merged**